src/Symfony/Component/Messenger/Middleware/TraceableMiddleware.php

```diff
@@ -72,7 +72,11 @@ public function __construct(StackInterface $stack, Stopwatch $stopwatch, string
     public function next(): MiddlewareInterface
     {
         if (null !== $this->currentEvent) {
```

Comment:
`if (null !== $this->currentEvent && $this->stopwatch->isStarted($this->currentEvent)) {`
like in `stop()`

Reply:
Good idea 👍🏼 
I fixed that.